.github/workflows/ci.yaml

```diff
@@ -7,15 +8,58 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: '3.8'
-          architecture: 'x64' # optional x64 or x86. Defaults to x64 if not specified
       - run: pip install poetry
       - run: poetry install
       - name: Log into Dockerhub
-        if: github.event_name == 'push'
+        if: github.event_name == 'push' && github.repository_owner == 'flavie'
```

Comment:
This was added so I can test my changes through my own fork's actions. But I would prefer if you could enable Actions for contributors at https://github.com/lavie/runlike/settings/actions.

![image](https://user-images.githubusercontent.com/12360257/221383108-b6957cf6-508d-46ea-9864-d7f249b17581.png)

I didn't have any problems yet with any contributors misusing the Github workflows.